### PR TITLE
ci(snap): enable arm build

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**

### CODE REVIEW

**Review by giusebar — APPROVED** [review]

**Comment by Guillaumebeuzeboc** [comment]
This PR need to migrate the tests to multi arch before getting merge

**Comment by Guillaumebeuzeboc** [comment]
Tests won't be ported to arm64 yet since the robotics-actions-workflow needs a major rework to be more flexible.

Here the issue is that we must build 3 snaps, and then run one test with these 3 snaps (and all of this on 2 different architecture).
Additionally, here there is an issue that the snap will get uploaded before getting tested.

We can merge this in the meantime to keep the active snap up to date.

**Review by artivis — APPROVED** [review]
LGTM :+1: .
But let's wait for the workflow refactoring to land before merging this.

**Review by artivis — APPROVED** [review]

**Pull request merged**
